.circleci/src/jobs/deliver_administration.yml

```diff
@@ -0,0 +1,44 @@
+docker:
+  - image: cimg/base:2022.09
+parameters:
+  production:
+    description: Whether builds are delivered to production or beta.
+    type: boolean
+steps:
+  - run:
+      name: Install Curl
+      command: |
+        sudo apt update
+        sudo apt install curl -y
+  - attach_workspace:
+      at: /tmp/workspace
```

Comment:
This is the wrong directory for the workspace. we use `~/attached_workspace` for it. Just use the`prepare_workspace` command and we can always be sure to use the correct folder.